Makefile.build

```diff
@@ -765,6 +765,21 @@ $(VDFILE).d: $(D_DEPEND_BEFORE_SRC) $(VFILES) $(D_DEPEND_AFTER_SRC) $(COQDEPBOOT
 	$(SHOW)'COQDEP    VFILES'
 	$(HIDE)$(COQDEPBOOT) -boot $(DYNDEP) $(VFILES) $(TOTARGET)
 
+###########################################################################
+# Bisect Coverage
+###########################################################################
+.PHONY: bisect_report
+
+bisect_report: coveralls.json
+
+BISECT_FILES=bisect*.out
+# use ocveralls --send ?
+coveralls.json: OCAMLC := $(OCAMLFIND) ocamlc $(CAMLFLAGS) -package bisect_ppx
+coveralls.json: OCAMLOPT := $(OCAMLFIND) opt $(CAMLFLAGS) -package bisect_ppx
```

Comment:
Why not use a configure option? Since it requires a rebuild to change that seems like the appropriate place to do it.

Reply:
I am moving away from configure, which is a big pain, in the future this will be place under a "workspace".

Reply:
Note that indeed this kind of package dependency cannot be solved by make or configure in a proper way.

Reply:
>Note that indeed this kind of package dependency cannot be solved by make or configure in a proper way.

make `configure -bisect-ppx` add the flag to CAMLFLAGS, problem solved no?

Reply:
That seems overkill.

Reply:
>That seems overkill.

It seems like the right way to change the compilation.
Also allows me to easily run with bisect-ppx on random developments locally.

Reply:
Yup you are right, so once I have a minute I'll do a patch adding `-camlflags -camlbyteflags -camloptflags` to `configure` and rebase this "patch" on top of it. [So indeed there won't be a `-bisect ppx` option, but Travis will just do `-camlflags "-package ppx_bisect"`.

Reply:
Feel free to take over that configure patch if you want; I am not gonna do it immediately, I just pushed this PR as not to let the tree bitrot on my HD.

Reply:
Never mind, I am finally doing this now.